arbor/backends/gpu/stack.hpp

```diff
@@ -56,7 +58,9 @@ class stack {
         return *this;
     }
 
-    explicit stack(unsigned capacity): storage_(create_storage(capacity)) {}
+    explicit stack(unsigned capacity, unsigned has_concurrent_managed_access):
```

Comment:
That `unsigned has_concurrent_managed_access` should probably be a `bool`.

Reply:
Thanks for pointing it out!